clang/test/dpct/asm/ld.cu

```diff
@@ -8,7 +8,7 @@
 #include <cuda_runtime.h>
 
 /*
-.ss =                       { .const, .global, .local, .param, .shared };
+.ss =                       { .const, .global, .local, .param, .shared, .volatile.global };
```

Comment:
According to the PTX spec: https://docs.nvidia.com/cuda/parallel-thread-execution/#data-movement-and-conversion-instructions-ld
The .ss field for the ld instruction does not contain ".volatile.global" value, it is invalid here.